platforms/linux/haxm-uninstall.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+DEVNAME=HAX
+GRPNAME=haxm
+MODNAME=haxm
+
+# Remove udev rule
+rm -f /lib/udev/rules.d/99-haxm.rules
+
+# Remove group, if necessary
+if [ $(getent group $GRPNAME) ]; then
+    groupdel -f $GRPNAME
```

Comment:
I'm on Ubuntu 16.04, and my `groupdel` command does not accept the `-f` flag:

```
groupdel: invalid option -- 'f'
Usage: groupdel [options] GROUP
```

Reply:
Turns out this flag was recently added, but it's not really necessary for us.
I will remove it in my next commit.